src/genie/libs/parser/iosxe/tests/ShowGnxiState/cli/equal/golden_output1_state_down_arguments.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Not necessary to include if it's empty

Reply:
Not necessary to use empty braces or not necessary to have the file?

Reply:
Not necessary to have the file. It's not a blocker, but it's nice to cut down on files we don't need